new-website/deepchem/components/Layers/FilterButton.js

```diff
@@ -0,0 +1,53 @@
+import Image from "next/image";
```

Comment:
This component is also exactly the same as `Models/FilterButton.jsx`. Please reuse the component, maybe move it to a directory named `Common` and use it in both models and layers page.

Reply:
done.